test/SendRecv_OutOfPlace.cpp

```diff
@@ -0,0 +1,90 @@
+/*************************************************************************
+ * Copyright (c) 2022 Advanced Micro Devices, Inc. All rights reserved.
+ *
+ * See LICENSE.txt for license information
+ ************************************************************************/
+#include "TestBed.hpp"
+
+namespace RcclUnitTesting
+{
+  TEST(SendRecv, OutOfPlace)
+  {
+    TestBed testBed;
+
+    // Configuration
+    std::vector<ncclFunc_t>     const  funcType        = {ncclCollSend, ncclCollRecv}; // akollias
+    std::vector<ncclDataType_t> const& dataTypes       = {ncclInt32};
+    std::vector<ncclRedOp_t>    const& redOps          = {ncclSum}; //Not important for send receive tests
+    std::vector<int>            const  numElements     = {1024}; // akollias, one test for now
+    bool                        const  inPlace         = false;
+    bool                        const  useManagedMem   = false;
+    int                         const  numCollPerGroup = numElements.size();

```

Comment:
For this test, you won't actually be using numCollPerGroup.  You will need it for SendRecv_AllPairs instead